src/python/grpcio/grpc/BUILD.bazel

```diff
@@ -15,16 +15,25 @@ py_library(
         ":interceptor",
         ":server",
         ":compression",
+        ":importer",
         "//src/python/grpcio/grpc/_cython:cygrpc",
         "//src/python/grpcio/grpc/experimental",
         "//src/python/grpcio/grpc/framework",
-        "@six_archive//:six",
+        "@six//:six",
     ] + select({
         "//conditions:default": ["@enum34//:enum34"],
         "//:python3": [],
     }),
 )
 
+py_library(
+    name = "importer",
+    srcs = [
+      "_importer.py",
+      "_service_reflection.py",
+    ],
+)
```

Comment:
Should we use two `py_library` rules instead of one?

Reply:
I don't think that's actually useful. These two files shouldn't be used separately from one another. I think it makes sense for them to be grouped.